src/Control/SAMLController.php

```diff
@@ -183,7 +182,7 @@ public function acs()
     public function metadata()
     {
         try {
-            /** @var OneLogin_Saml2_Auth $auth */
+            /** @var OneLogin\Saml2\Auth $auth */
```

Comment:
As earlier, the namespace is unnecessary, as the class is imported above (`use ...`)